values.yaml

```diff
@@ -1 +1,24 @@
 ---
+images:
+  gitInit: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:latest
+
+annotations:
+  tekton.dev/pipelines.minVersion: "0.38.0"
+  tekton.dev/categories: Git
+  tekton.dev/tags: git
+  tekton.dev/displayName: "git"
+  tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le,linux/arm64"
+
+stepTemplate:
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 65532
+  resources:
+    limits:
+      cpu: 100m
+      memory: 256Mi
+    requests:
+      cpu: 100m
+      memory: 256Mi
```

Comment:
This needs to reflect the current project.

Reply:
Ok I will update it.